care/facility/models/patient_icmr.py

```diff
@@ -138,7 +138,9 @@ def lab_pincode(self):
 
     @property
     def hospitalization_date(self):
-        return self.consultation.admission_date.date()
+        return (
+            self.consultation.admission_date.date() if self.consultation and self.consultation.admission_date else None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Instance of 'ForeignKey' has no 'admission_date' member](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5348668)